infra/gcp/ensure-staging-storage.sh

```diff
@@ -56,6 +56,7 @@ STAGING_PROJECTS=(
     multitenancy
     publishing-bot
     release-test
+    kube-state-metrics
```

Comment:
@brancz I think you need to maintain lexical order here. As stated in the comment above.

Reply:
Done